### DISCUSSION TITLE
I want to use the Nginx proxy service port to access the service through a domain name. How can I configure it?

### DISCUSSION THREAD

**Discussion by PineSongCN** [rustdesk/rustdesk#9332] [Q&A]
I want to use the Nginx proxy service port to access the service through a domain name. How can I configure it?
> 我想使用nginx代理服务端口，通过域名访问服务，请问该如何配置？

**Comment by fergalc** [comment]
I am using [RustDeskServer-AiO Docker](https://hub.docker.com/r/ich777/rustdesk-server-aio) (192.168.1.2) and have it working behind Nginx Proxy Manager Docker (192.168.1.1) self hosted. I use Cloudflare and have my own domain (selfhosted.com) pointing to my external IP. My firewall has ports 80 & 443 opened to 192.168.1.1
Once you have NPM talking to Cloudflare everything will work accordingly.
In NPM I have a proxy for rustdesk.selfhosted.com pointing to 192.168.1.2:21117
I also have in NPM TCP Streams created for ports 211175 - 211179 pointing to 192.168.1.2
I have a UDP Stream created for 21116 pointing to 192.168.1.2
I have the above ports also opened in my firewall pointing to 192.168.1.2.

**Reply by rustdesk** [reply]
This image is not maintained by RustDesk, please take your risk to use it.

**Reply by fergalc** [reply]
Thank you for the heads up.

**Comment by FloydKulis** [comment]
@fergalc 
How did you manage to do this? He wants to add a domain via cloudflare and add a redirection in NGINX. Unfortunately, in the program itself I can't enter it:
<img width="409" alt="image" src="https://github.com/user-attachments/assets/7356616a-d5fb-42d7-8076-c2d9d79b0dce" />
